Author: Guido Guenther <agx@sigxcpu.org>
Description:
 Set default framebuffer device to /dev/fb0 instead of /dev/fb which doesn't
 exist by default (Closes: #573724).

diff -Naurp plymouth.orig/src/plugins/renderers/frame-buffer/plugin.c plymouth/src/plugins/renderers/frame-buffer/plugin.c
--- plymouth.orig/src/plugins/renderers/frame-buffer/plugin.c	2010-01-02 11:22:54.000000000 +0100
+++ plymouth/src/plugins/renderers/frame-buffer/plugin.c	2010-03-14 17:14:49.000000000 +0100
@@ -46,7 +46,7 @@
 #include <linux/fb.h>
 
 #ifndef PLY_FRAME_BUFFER_DEFAULT_FB_DEVICE_NAME
-#define PLY_FRAME_BUFFER_DEFAULT_FB_DEVICE_NAME "/dev/fb"
+#define PLY_FRAME_BUFFER_DEFAULT_FB_DEVICE_NAME "/dev/fb0"
 #endif
 
 struct _ply_frame_buffer
